client/src/components/ProjectList/ProjectList.jsx

```diff
@@ -1,56 +1,52 @@
-import React, { useEffect, useState } from 'react';
-import axios from 'axios';
+import React, {useEffect, useState} from 'react';
 import styles from './ProjectList.module.scss';
 import EmptyProjectList from './EmptyProjectList';
 import ProjectListItem from './ProjectListItem/ProjectListItem';
-// import projectList from '../../consts/projectList';
+import {useHttp} from '../../hooks/http_hook';
```

Comment:
Please, dont use snake case in imports.
https://github.com/airbnb/javascript#naming--filename-matches-export